### PR TITLE
Database query update

### PR DESCRIPTION
**Pull request by efoosag**
- Use database transactions.
- Modify and delete data in SQL.
- Query with aggregate functions

Snapshot
transaction1
transaction2
transaction3
transaction4

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -7,3 +7,5 @@ CREATE TABLE ANIMALS (
     ESCAPE_ATTEMPTS INT, 
     NEUTERED BOOLEAN,
     WEIGHT_KG DECIMAL);
+
+ALTER TABLE animals ADD species VARCHAR(50);
```
Comment (mateo951) [inline schema.sql:11]: Awesome 🚀 Well done!

File: data.sql [hunk 2]
```diff
@@ -10,4 +10,25 @@ INSERT INTO ANIMALS (id, name, date_of_birth, escape_attempts, neutered, weight_
...
+VALUES(8, 'Angemon','2005-06-12', 1, true, 45);
+
+INSERT INTO ANIMALS (id, name, date_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES(9, 'Boarmon','2005-06-07', 7, true, 20.4);
+
+INSERT INTO ANIMALS (id, name, date_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES(10, 'Blossom','1998-10-13', 3, true, 17);
+
+INSERT INTO ANIMALS (id, name, date_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES(11, 'Ditto','2022-05-14', 4, true, 22);
```
Comment (mateo951) [inline data.sql:34]: Perfect 💪 Correct data was introduced 😎

File: queries.sql [hunk 3]
```diff
@@ -15,3 +15,15 @@ SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name != 'Gabumon';
 
 SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
```
Comment (mateo951) [inline queries.sql:18]: Please be sure to add the four transactions that the requirements mention 🤓 

For example for the first transaction:

```sql
   BEGIN; -- start transaction

   UPDATE animals
   SET species = 'unspecified'; -- make change

   SELECT species from animals; -- verify that change was made
   ROLLBACK;

   SELECT species from animals; -- verify that change was undone
```

**Review by mateo951 — CHANGES_REQUESTED** [review]
### Hi @efoosag, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Review by Davidosky007 — APPROVED** [review]
Hi @efoosag 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
* Proper use of Gitflow 👍 
* Well written PR description 👍 
* All transactions written properly 👏 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**